Skype/Trusted-Application-API/docs/AnonymousMeetingJoin.md

```diff
@@ -28,9 +28,9 @@ In [Registering your application in Azure AD](RegistrationInAzureActiveDirectory
 - Create on-demand Skype meetings (preview)
 - Send/Receive Instant Messages (preview)
 
-![alt text](./images/GuestMeetingJoinTenantConsent.png "image") 
+![Screenshot of the required permissions for Guest user join services (preview), Send/Receive Audio and Video (preview), Create on-demand Skype meetings (preview), Send/Receive Instant Messages (preview) and an accept or cancel button choice.](./images/GuestMeetingJoinTenantConsent.png "image") 
```

Comment:
Since this is alt text, we have to remember to come at these solutions from the perspective of how a screen reader would interpret the words and characters.  How about something like:
"Screenshot of the required permissions for Guest user join services preview, Send Receive Audio and Video preview, Create on demand Skype meetings 
 preview, Send Receive Instant Messages preview and an accept or cancel button choice."